physical/datasource.go

```diff
@@ -64,9 +65,10 @@ type DataSourceBuilder struct {
 	Filter           Formula
 	Name             string
 	Alias            string
+	Cardinality      metadata.Cardinality
```

Comment:
You could add a short comment to describe what the goal of this field is

Reply:
Yea, to make the naming consistent.